examples/proc_capture_nutation.py

```diff
@@ -1,4 +1,8 @@
-""" Processes and visualizes the nutation pulse program that has been 
+"""
+title needed!!
+==============
```

Comment:
the ==== is an rst thing that indicates the title, and this is needed --> there are a couple of these